design-editor/src/panel/property/component/component-element.js

```diff
@@ -312,42 +306,46 @@ class Component extends DressElement {
         });
     }
 
-    /**
-     * All Component should be had own Icon that display on component tab.
-     * @param {string} $itemList
-     * @private
-     */
-	_setIcon($itemList = $([]) /* Compnent list*/) {
-        var name,
-            options,
-            imgPath;
-
-        $itemList.each((index, element) => {
-            let icon = null;
-
-            name = $(element).data('component-name');
-            options = componentsInfo[name].options;
-            icon = options.resources.icon;
-
-            if (typeof icon !== 'string' && icon) {
-                icon = icon[this._profile];
-                if (typeof icon !== 'string' && icon) {
-                    icon = icon[this._shape];
-                }
-            }
-
-            if (typeof icon === 'string' && icon) {
-                imgPath = path.join(options.path, icon);
-                // on Windows path separator is \, having this not encoded in css url is invalid and images are not loaded
-                imgPath = imgPath.replace(/\\/g, '/');
-                $(element).css('background-image', 'url("' + imgPath + '")')
-                    .removeClass('closet-component-element-list-item-noicon');
-            } else {
-                $(element).css('background-image', 'none')
-                    .addClass('closet-component-element-list-item-noicon');
-            }
-        });
-    }
+	/**
+	 * All Component should be had own Icon that display on component tab.
+	 * @param {Element[]} $itemList -Compnent list
+	 * @private
+	 */
+	_setIcon(itemList) {
+
+		itemList = itemList || [];
```

Comment:
Is that any reason to use here OR syntax instead of default param?

Reply:
yes - this default patters catches only undefined. Null value will pass unchanged.